bears/python/PycodestyleBear.py

```diff
@@ -0,0 +1,48 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='pycodestyle',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+) (?P<column>\d+) (?P<message>.*)')
+class PycodestyleBear:
+    """
+    A wrapper for the tool ``pycodestyle`` formerly known as ``pep8``.
+    """
+    LANGUAGES = {"Python", "Python 2", "Python 3"}
```

Comment:
do we need to run pycodestyle with the python 2/3 binary to get python 2 or 3 support?